content/api/alerts.apib

```diff
@@ -1,205 +1,322 @@
 FORMAT: 1A
 title: Alerts
-description: Configure alerts for your SparkPost account.
-label: Labs
+description: Configure real-time alerts for your SparkPost account.
+label: New
 
 # Group Alerts
-Alerts can be used to get notifications regarding various activities on your SparkPost account.
+Alerts can be used to get notifications regarding various activities and metrics on your SparkPost account.
 
-Alerts can be configured to send notifications via email to up to 10 email addresses, and supports the following activity:
-* Approaching monthly sending limit
+## Metrics
+At this time, the following metrics are supported:
 
-Alerts are configured to be account-wide. Currently, subaccount level alerts are not supported. 
+* Real-Time
+    + monthly_sending_limit
+    + block_bounce_rate
+* Signals
+    + health_score
+
+Each metric group has a specific set of filters and some other values allowed.
+
+* Real-Time
+    + filters - filter_type can be 'sending_domain', 'sending_ip', or 'mailbox_provider'
+    + threshold_evaluator - source must be 'raw'
+* Signals
+    + filters - filter_type can be 'sending_domain', 'ip_pool', or 'mailbox_provider'. Only one filter type can be selected,
+        though multiple values are allowed.
+
+* monthly_sending_limit
+  + filters - must be an empty array. No filters may be applied to this metric.
+  + subaccount - must be 0.
```

Comment:
subaccount `0` is different than all sending on an account.

Reply:
Agreed on a new phrasing/structure